cloud_images/centos7/dcos_vol_setup.sh

```diff
@@ -35,23 +35,48 @@ function main {
     exit 1
   fi
 
+  echo "Waiting for $device to come online"
+  until test -b "$device"; do sleep 1; done
```

Comment:
can we sleep and echo here (that way get sort of a log how long it takes)?

Same goes for the other 'Waiting for'.